src/viam/examples/modules/complex/proto/buf.lock

```diff
@@ -4,5 +4,5 @@ deps:
   - remote: buf.build
     owner: googleapis
     repository: googleapis
-    commit: 7e6f6e774e29406da95bd61cdcdbc8bc
-    digest: shake256:fe43dd2265ea0c07d76bd925eeba612667cf4c948d2ce53d6e367e1b4b3cb5fa69a51e6acb1a6a50d32f894f054a35e6c0406f6808a483f2752e10c866ffbf73
+    commit: ee48893a270147348e3edc6c1a03de0e
```

Comment:
I'm actually not sure what this file does but it got changed as a side effect of building so I've included it. Please let me know if I should revert.